src/js/providers/video-actions-mixin.js

```diff
@@ -1,4 +1,7 @@
 import { style } from 'utils/css';
+import fitToBounds from 'utils/fit-to-bounds';
+import fitVideoUsingTransforms from 'utils/fit-to-bounds';
```

Comment:
This does not do what you think it does. You need curly braces in order to import a module that's not the default exported module, so: 

`import { fitVideoUsingTransforms } from 'utils/fit-to-bounds';`